tensorflow/workspace.bzl

```diff
@@ -83,6 +84,49 @@ temp_workaround_http_archive = repository_rule(
 )
 
 
+def _run_cmd(repo_ctx, cmd, check=True):
+  if _is_windows(repo_ctx):
+    bazel_sh = _get_env_var(repo_ctx, "BAZEL_SH")
+    if not bazel_sh:
+      fail("BAZEL_SH environment variable is not set")
+    cmd = [bazel_sh, "-c", " ".join(cmd)]
+  if check:
+    _execute_and_check_ret_code(repo_ctx, cmd)
+  else:
+    return repo_ctx.execute(cmd, timeout=10)
+
+def _unzip(repo_ctx, zip_file, strip_prefix):
+  _run_cmd(repo_ctx,
+           ["unzip", repo_ctx.path(zip_file),
```

Comment:
(anyway; i'm pretty sure that the crosstool compiler won't run on windows anyway...)